subject-similarity-groups/src/test/scala/edu/gatech/cse6250/graphconstruct/GraphLoaderTest.scala

```diff
@@ -0,0 +1,234 @@
+package edu.gatech.cse6250.graphconstruct
```

Comment:
We should probably just remove all of these test files

Reply:
done